File: Programmers/Stack, Queue/기능개발.py
import math

def solution(progresses, speeds):
    answer = []
    progress = [math.ceil((100 - progresses[i]) / speeds[i]) for i in range(len(progresses))]
    
    idx = 0
    for i in range(len(progress)):  
        if progress[i] > progress[idx]:
            answer.append(i-idx)
            idx = i
    answer.append(len(progress) - idx)
    return answer

File: Programmers/Stack, Queue/기능개발.js
https://programmers.co.kr/learn/courses/30/lessons/42586?language=javascript

function solution(progresses, speeds) {
    var answer = [];

    let remain = [];
    for (let i = 0; i < progresses.length; i++) {
        remain.push(Math.ceil((100 - progresses[i]) / speeds[i]));
    }

    let idx = 0;
    for (let i = 0; i < progresses.length; i++) {
        if (remain[i] > remain[idx]) {
            answer.push(i - idx);
            idx = i;
        }
    }
    answer.push(remain.length - idx);

    return answer;
}